### ISSUE TITLE
Duplicate case created from system mailbox

### ISSUE THREAD

**Issue by ianfp** [orocrm/crm#150]
I'm using OroCRM 1.9.0-RC2. I've added a system mailbox and configured it to use IMAP and automatically create cases from emails that end up in the "Webmaster" folder. I'm using Gmail (Google Apps for business, specifically) as the IMAP server for this test. I've configured a Gmail filter to automatically move emails to `webmaster@...` to the "Webmaster" folder/label.

Here's the problem: I sent an email to `webmaster@...` and then ran the `oro:cron:imap-sync` and `jms-job-queue:run` console commands. When I look at the list of cases, I now have not one but two identical new cases created from that email.

**Comment by dekey** [comment]
Hi @ianfp. Could you please specify your system mailbox settings and how many system mailbox do you have? If you have more than one system box, do they have redirect from one to another?

**Comment by ianfp** [comment]
@dekey, thanks for following up.

It is a fresh install with an empty database, and I've added only one system mailbox. (None of the users have any personal mailboxes set up.)

The mailbox settings are:
Label: webmaster@mydomain.com
Email: webmaster@mydomain.com

Enable imap: true
IMAP Host: imap.gmail.com
IMAP Port: 993
Encryption: SSL

Enable SMTP: false

User: me@mydomain.com
Password: XXXX

Some notes: 1) I've selected only the "Webmaster" folder from the list of IMAP folders. 2) webmaster@mydomain.com is an alias to me@mydomain.com. Any email sent to webmaster gets filtered by Gmail and automatically put in the "Webmaster" folder.

I hope that helps!
